### ISSUE TITLE
VaHealthCareLocalFacilityStatusMigrationTest fails on subsequent runs

### ISSUE THREAD

**Issue by acrollet** [department-of-veterans-affairs/va.gov-cms#4638]
**Describe the defect**

This is a follow-on from #4349. The VaHealthCareLocalFacilityStatusMigrationTest test fails on runs after the first one. (Likely because a node is not getting correctly deleted after the first test.)

**To Reproduce**
1. Run `lando phpunit-run tests/phpunit/Migration/VaHealthCareLocalFacilityStatusMigrationTest.php`
2. The test should succeed
3. Run `lando phpunit-run tests/phpunit/Migration/VaHealthCareLocalFacilityStatusMigrationTest.php` again
2. Test fails with:
```
1) tests\phpunit\Migration\VaHealthCareLocalFacilityStatusMigrationTest::testVaLocalHealthCareFacilityStatusMigration with data set "Initial migration completes successfully" ('va_node_health_care_local_fac...status', 'health_care_local_facility', '\n<!DOCTYPE html>\n<head><tit...tml>\n', array('vha_999999', 'normal', 'All Veteran patients are ente...place.'), 1, false)
Failed asserting that 0 is identical to 1.
```

**Expected behavior**
Test should succeed on multiple runs.

**Comment by acrollet** [comment]
#4714 resolves this issue.

**Comment by acrollet** [comment]
Fixed by #4714
